.github/workflows/test-count.yml

```diff
@@ -0,0 +1,52 @@
+name: TCK Test Count
+
+on:
+  pull_request:
+    paths:
+      - "docs/test-specifications/**"
+      - "src/tests/**"
```

Comment:
Do we need this to run on code changes? Since it only determines if a test is implemented from changes in the docs?

Reply:
true i can take that out